CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Add OkHttp event spans ([#2659](https://github.com/getsentry/sentry-java/pull/2659))
```

Comment:
`m`: Could make the changelog more descriptive? Maybe something like

```suggestion
- Instrument http requests more granularly for OkHttp integration ([#2659](https://github.com/getsentry/sentry-java/pull/2659))
  - Create spans for time spent on:
  	- DNS resolution
  	- Connecting
  	- Requesting headers
  	- ...
```

Reply:
M: Also add code snippets on how to set up the listener.

Reply:
Code snippet is in the Javadoc of the Listener. Is it enough there, or should I add it in the changelog, too?